cmd/satellite/main.go

```diff
@@ -60,9 +60,9 @@ var (
 		Short: "Generate a report",
 	}
 	paymentsCmd = &cobra.Command{
```

Comment:
maybe we should change variable names too?

Reply:
yeah i was wondering that...if all the "payment" names should be updated to "usage" stuff.